testsuite/setup.py

```diff
@@ -115,7 +115,7 @@
                   'data/*.trz',
                   'data/mol2/*.mol2',
                   'data/capping/*.gro', 'data/capping/*.pdb',
-                  'data/*.data',
+                  'data/lammps/*.data', 'data/lammps/*.data.bz2', 'data/lammps/*.dcd', 'lammps/*.trz',
```

Comment:
No, in file is not needed to run the tests, just in case we want to rerun LAMMPS again to generate a DCD, eg with different settings. 

--
Oliver Beckstein
email: orbeckst@gmail.com
skype: orbeckst


Am Oct 6, 2015 um 9:49 schrieb Richard Gowers <notifications@github.com>:

> In testsuite/setup.py:
> 
> > @@ -115,7 +115,7 @@
> >                    'data/*.trz',
> >                    'data/mol2/*.mol2',
> >                    'data/capping/*.gro', 'data/capping/*.pdb',
> > -                  'data/*.data',
> > +                  'data/lammps/*.data', 'data/lammps/*.data.bz2', 'data/lammps/*.dcd', 'lammps/*.trz',
> the .in file needs to be here too?
> 
> —
> Reply to this email directly or view it on GitHub.
>